crates/router/src/connector/multisafepay/transformers.rs

```diff
@@ -390,7 +395,7 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for MultisafepayPaymentsReques
             recurring_model: if item.request.setup_future_usage
                 == Some(enums::FutureUsage::OffSession)
```

Comment:
recurring_model should be based on setup_mandate_details and not item.request.setup_future_usage